frontend/src/app/reducers/varianttests.js

```diff
@@ -38,9 +38,34 @@ const tests = [
 const chosenVariants = {};
 const identityReducers = {};
 
+let chosenTest = null;
+
 tests.forEach(test => {
-  chosenVariants[test.name] = test.getValue();
+  // Only put each user in one test. If we previously found a non-default test,
+  // put this user in default for the rest of the tests.
```

Comment:
I'm not saying this is wrong, but it makes me wonder how the order of `tests` affects the populations of users they have access to. For example, 2 tests with a 50% chance on a variant. The first test will get 50% of all users as expected, but the second only 25% (50% of the remaining 50%). Am I right? Does that matter? I'm certainly no expert.